projects/dex-ui/src/pages/Well.tsx

```diff
@@ -35,8 +36,8 @@ export const Well = () => {
       <strong>Reserves: </strong>
       {well.reserves?.map((r, i) => `${r.toHuman("0,0.00a")} ${well.tokens?.[i].symbol}`).join(" - ")}
       <br />
-      <div>{isLoadingAllBalances ? <div>Spinner</div> : <AddLiquidity well={well} txnCompleteCallback={liquidityTxnCallback} />}</div>
-      <div>{isLoadingAllBalances ? <div>Spinner</div> : <RemoveLiquidity well={well} txnCompleteCallback={liquidityTxnCallback} />}</div>
+      <br />
+      <div>{isLoadingAllBalances ? <div>SPINNER IMAGE</div> : <LiquidityRoot well={well} txnCompleteCallback={liquidityTxnCallback} />}</div>
```

Comment:
we have a loader component, <Spinner size={32} />

Reply:
✅